src/types/langs.ts

```diff
@@ -1,12 +1,12 @@
-type SignUpPageType = {
-  Main: {
-    [key: string]: string;
-  };
-  TermsContent: {
-    [key: string]: string;
-  };
-};
+// type SignUpPageType = {
+//   Main: {
+//     [key: string]: string;
+//   };
+//   TermsContent: {
+//     [key: string]: string;
+//   };
+// };
 
```

Comment:
delete commented code if it's not part of documentation

Reply:
fixed